pkg/progressreader/progressreader.go

```diff
@@ -11,15 +11,16 @@ import (
 
 // Config contains the configuration for a Reader with progress bar.
 type Config struct {
-	In         io.ReadCloser // Stream to read from
-	Out        io.Writer     // Where to send progress bar to
-	Formatter  *streamformatter.StreamFormatter
-	Size       int64
-	Current    int64
-	LastUpdate int64
-	NewLines   bool
-	ID         string
-	Action     string
+	In             io.ReadCloser // Stream to read from
+	Out            io.Writer     // Where to send progress bar to
+	Formatter      *streamformatter.StreamFormatter
+	Size           int64
+	Current        int64
+	LastUpdate     int64
+	NewLines       bool
+	NewLineInClose bool
```

Comment:
Instead of adding another option to `ProgressReader` that's specific to this use case, can you do something like this in `copyFromContainer`:

```go
defer func() {
        content.Close()
        // Format blank line
        cli.out.Write(sf.FormatStatus("", ""))
}()
```